### ISSUE TITLE
Idea: Move treeMenu into tab "Site Tree"

### ISSUE THREAD

**Issue by Deesen** [pmfx/ElementsInTree#8]
Maybe inspiring.. with a simple

    jQuery("#treeMenu").detach().prependTo("#tabDoc");

after

    jQuery(document).ready(function() {

it is possible to move the tree-menu like

![treemenu](https://cloud.githubusercontent.com/assets/8569221/19636403/75dafba4-99c9-11e6-9ae2-581981b17200.png)

**Comment by pmfx** [comment]
Good idea, makes sense as ~75% of buttons there are tree related. Do you plan to do it? I can work on this if you want.

**Comment by pmfx** [comment]
Maybe having it as an option (turned on by default) would be good.

**Comment by Deesen** [comment]
For me it makes sense, but not sure what others think about it.

**Comment by Deesen** [comment]
And no plans from my side, please feel free..

**Comment by pmfx** [comment]
Done. Can you test it?
![screenshot_1](https://cloud.githubusercontent.com/assets/10888055/19640064/6628d9f8-99db-11e6-9b2d-20c8557b86b7.png)

**Comment by pmfx** [comment]
Style without background proposition:

![screenshot_2](https://cloud.githubusercontent.com/assets/10888055/19640421/e1036976-99dc-11e6-9ecc-49112b28f35e.png)

Original:

![screenshot_3](https://cloud.githubusercontent.com/assets/10888055/19640443/fb3bab50-99dc-11e6-841d-1a951504e3b1.png)

What do you think?

**Comment by pmfx** [comment]
After 15 minutes of staring at it, I lean more towards original ;)

**Comment by fourroses666** [comment]
Good to place it into the tree, I like the transparent bg.

**Comment by pmfx** [comment]
@fourroses666 After another 15 minutes I think you're right. Transparent it is.

**Comment by Deesen** [comment]
Tested works for me. Not sure about the layout. I tried something like this but its toooo solid LOL:

![treemenu2](https://cloud.githubusercontent.com/assets/8569221/19643290/dde65bf6-99e9-11e6-9d66-a17f78ab9426.png)

**Comment by pmfx** [comment]
Yeah little too bold. Lets stay with transparent for now. Closing issue. Thanks Deesen for this nice idea!
